Documentation/frontera.rst

```diff
@@ -81,3 +82,74 @@ To access the module at start up, add to your ``~/.bashrc``:
     module use $SPACK_ROOT/share/spack/modules/linux-centos7-cascadelake/
 
 Finally, install SeisSol with cmake, as usual, with ``-DHOST_ARCH=skx``.
+
+For large runs, it is recommanded to have executable, dynamic libraries, setup and outputs on scratch.
+That is, $WORK is not built for large, intensive IOs, and loading the shared libraries from 8000+ nodes at once is quite intensive.
+This could potentially break the filesystem.
+The dynamic libraries can be copied to $SCRATCH with the following commands:
+
+.. code-block:: bash
+
+    # replace by the path to your seissol executable
+    mkdir -p $SCRATCH/libdump  && ldd SeisSol_Release_dskx_6_elastic | grep -E "/work|/scratch" | awk '{print $(NF-1)}' | xargs -I _ cp _ $SCRATCH/libdump
+
+Then you can unload the seissol-env module and add the required dynamic libraries, e.g. with:
+
+.. code-block:: bash
+
+    export LD_LIBRARY_PATH=$SCRATCH/lib_dump/:$LD_LIBRARY_PATH
+    module unload seissol-env
+
+Finally, we provide an example of launch script used for running a full-machine frontera run.
+In particular, note how timeout and retry count are increased.
+
+.. code-block:: bash
+
+    #!/bin/bash
+    #SBATCH --chdir=./
+    #SBATCH -o ./%j.out       # Name of stdout output file
+    #SBATCH -e ./%j.out       # Name of stderr error file
+    #SBATCH -p debug         # Queue (partition) name
+    #SBATCH --nodes=8192
+    #SBATCH --ntasks-per-node=2
+    #SBATCH -t 24:00:00        # Run time (hh:mm:ss)
+    #SBATCH -A EAR22007       # Project/Allocation name (req'd if you have more than 1)
+
+    # Any other commands must follow all #SBATCH directives...
+    module list
+    pwd
+    date
+
+    #Prevents errors such as experience in Issue #691
+    export I_MPI_SHM_HEAP_VSIZE=32768
+
+    export OMP_NUM_THREADS=27
+    export OMP_PLACES="cores(27)"
+    #export OMP_PROC_BIND="close"
```

Comment:
Is this needed—or assumed to be true by default?

Reply:
it is half hte number of threads of a node, so I doubt it would be set by default, right?

Reply:
It's better to do it explicitly.